gtk/Utils.h

```diff
@@ -130,6 +132,14 @@ inline T gtr_str_strip(T const& text)
 
 std::string gtr_get_full_resource_path(std::string const& rel_path);
 
+/***
+****
+***/
+
+extern size_t const max_recent_dirs;
+std::list<std::string> gtr_get_recent_dirs(std::string const& pref);
```

Comment:
`#include <list>` needs to go to the top of this header

Reply:
Thanks! Header fixed in https://github.com/transmission/transmission/pull/3691/commits/24fc00776dc8ee3103feb669db38889b66d1023b.